srtcore/epoll.cpp

```diff
@@ -909,7 +911,7 @@ int CEPoll::update_events(const SRTSOCKET& uid, std::set<int>& eids, const int e
     for (vector<int>::iterator i = lost.begin(); i != lost.end(); ++ i)
         eids.erase(*i);
 
-    return 0;
+    return nupdated;
```

Comment:
Return values are changed.
Please update comments to the function in epoll.h.
```c++
   /// Update events available for a UDT socket.
   /// @param [in] uid UDT socket ID.
   /// @param [in] eids EPoll IDs to be set
   /// @param [in] events Combination of events to update
   /// @param [in] enable true -> enable, otherwise disable
   /// @return 0 if success, otherwise an error number

   int update_events(const SRTSOCKET& uid, std::set<int>& eids, int events, bool enable);
```